app/models/provider.rb

```diff
@@ -0,0 +1,15 @@
+class Provider < ApplicationRecord
+    has_many :ingredients, dependent: :destroy
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.